07-routing-gossip.md

```diff
@@ -727,6 +727,134 @@ is simple to implement.
 In the case where the `channel_announcement` is nonetheless missed,
 `query_short_channel_ids` can be used to retrieve it.
 
+## Extended Query Messages
+
+The query messages defined above do not cover the use case of nodes
+that are offline most of the time and are missing a timestamps that can
+be used to detect obsolete channel updates.
+
+Here we define a new set of channel queries, with additional timestamps (one
+timestamp per channel update).
+
+Encoding types are the same as above, with a single byte prefix (`0` or `1`)
+
+### The `query_short_channel_ids_with_timestamps`/`reply_short_channel_ids_with_timestamps_end` Messages
+
+1. type: 261 (`query_short_channel_ids_with_timestamps`) (`gossip_queries`)
```

Comment:
what's the rationale for breaking compatibility with older nodes, instead of allocating the 101x messages to the extended queries?

I think this merits it's own (`extended_gossip_queries`) feature bit, instead of overloading (`gossip_queries`)?

Reply:
Ah yes I messed up the new message types, it's fixed now

Reply:
And I've added a new feature bit, with the value we've used for our own tests